src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetCultureCommand.cs

```diff
@@ -1,23 +1,94 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.Globalization;
 using System.Management.Automation;
 
 namespace Microsoft.PowerShell.Commands
 {
     /// <summary>
-    /// Returns the thread's current culture.
+    /// Returns:
+    ///     - the thread's current culture
+    ///     - culture by name
+    ///     - list of all supported cultures.
     /// </summary>
-    [Cmdlet(VerbsCommon.Get, "Culture", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113312")]
+    [Cmdlet(VerbsCommon.Get, "Culture", DefaultParameterSetName = DefaultParameterSet, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113312")]
     [OutputType(typeof(System.Globalization.CultureInfo))]
     public sealed class GetCultureCommand : PSCmdlet
     {
+        private const string DefaultParameterSet = "Default";
+        private const string NameParameterSet = "Name";
+        private const string ListParameterSet = "List";
+
+        /// <summary>
+        /// Gets or sets culture names for which CultureInfo values are returned.
+        /// </summary>
+        [Parameter(ParameterSetName = NameParameterSet, Position = 0, ValueFromPipeline = true)]
+        [ValidateNotNull]
+        public string[] Name { get; set; }
+
         /// <summary>
-        /// Output the current Culture info object.
+        /// Gets or sets a switch to return predefined cultures.
+        /// By default we return cultures in current state (with custom changes).
+        /// With the switch on, we return predefined, original cultures.
         /// </summary>
-        protected override void BeginProcessing()
+        [Parameter(ParameterSetName = NameParameterSet)]
+        [ValidateNotNull]
+        public SwitchParameter Predefined { get; set; }
+
+        /// <summary>
+        /// Gets or sets a filter to list subset or all available cultures.
+        /// </summary>
+        [Parameter(ParameterSetName = ListParameterSet)]
+        public CultureTypes List { get; set; } = CultureTypes.AllCultures;
+
+        /// <summary>
+        /// Output:
+        ///     - the thread's current culture
+        ///     - culture by name
+        ///     - list of all supported cultures.
+        /// </summary>
+        protected override void ProcessRecord()
         {
-            WriteObject(Host.CurrentCulture);
+            switch (ParameterSetName)
+            {
+                case DefaultParameterSet:
+                    WriteObject(Host.CurrentCulture);
+
+                    break;
+                case NameParameterSet:
+                    try
+                    {
+                        foreach (var cultureName in Name)
+                        {
+                            CultureInfo ci;
+
+                            if (Predefined)
```

Comment:
As stated, I don't think we need this, after all - just always use `.GetCultureInfo()` to get the _original_ definition, without a user override (if the current culture happens to be requested).
(It is therefore a moot point, but just to be clear it is `new CultureInfo(cultureName, true)` (or just `new CultureInfo(cultureName)` that _does_ give you the overrides, so the logic here is backwards.)